### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by sahrulhs**

### CODE REVIEW

File: ex20.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from sys import argv
+
+script, input_file = argv
+
+def print_all(f):
+	print f.read()
+
```
Comment (ankit167) [inline ex20.py]: Please use two spaces between functions.

File: ex20.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+current_file = open(input_file)
+
+print "First let's print the whole file:\n"
+print_all(current_file)
+
+print "Now let's rewind, kind of like a tape."
+rewind(current_file)
+
+print "Let's print three lines:"
+current_line = 1
```
Comment (ankit167) [inline ex20.py]: Better to use a loop. Would reduce Lines of Code.

File: ex20.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	print f.read()
+
+# seek(n) to read a file's content from byte-n
+def rewind(f):
+	f.seek(0)
+
+def print_a_line(line_count, f):
+	print line_count, f.readline()
+
+current_file = open(input_file)
```
Comment (ankit167) [inline ex20.py]: Please wrap non-function code inside main()

Comment (mam8) [inline ex20.py]: If the **input_file** is not existing, the **open** will fail.

File: ex20.py (hunk 1 above)
Comment (sahrulhs) [inline ex20.py]: I type this code on debian's gedit, in gedit nothing automatic indentation, so the result like this...

File: ex20.py (hunk 3 above)
Comment (sahrulhs) [inline ex20.py]: This code run on command line, so must there is a file in same directories.

Comment (mam8) [inline ex20.py]: I am also a newbie. I just learned a good way for this. It's with ... as ...: 
with open( input_file) as current_file:

Comment (ankit167) [inline ex20.py]: @sahrulhs The code is generic. Ideally, it should handle every case, which includes the file not being present at all.

**Pull request merged**